synapse/federation/federation_server.py

```diff
@@ -1399,7 +1399,7 @@ async def on_edu(self, edu_type: str, origin: str, content: dict) -> None:
         # Check if we have a handler on this instance
         handler = self.edu_handlers.get(edu_type)
         if handler:
-            with start_active_span_from_edu(content, "handle_edu"):
+            with start_active_span_from_edu("handle_edu", edu_content=content):
```

Comment:
> AFAICT, this never worked because this PR serializes everything into `content["org.matrix.opentracing_context"]` but `start_active_span_from_edu` reads from `content["opentracing"]` (even at this point in time in the code to current day).
> 
> https://github.com/matrix-org/synapse/blob/bbc23cdd04a2dfafefcbcb3264c2e89b50eacc04/synapse/logging/opentracing.py#L402
> 
> *-- @MadLittleMods, https://github.com/matrix-org/synapse/pull/5852#discussion_r934960586*